### PR TITLE
Connector syslog watch guard firebox

### PR DESCRIPTION
**Pull request by ecosystempo**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Comment by ecosystempo** [comment]
Update WatchGuard Data Connector

File: DataConnectors/Connector_syslog_WatchGuardFirebox.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            "query": "Firebox\n | where HostName contains 'WatchGuard'\n| where TimeGenerated >= ago(24h)\n| summarize count() by SyslogMessage\n| top 10 by count_ desc"
+        },		
+		{
+            "description" : "Firebox top 10 HostIP in last 24 hours",
+            "query": "Firebox\n | where HostName contains 'WatchGuard'\n| where TimeGenerated >= ago(24h)\n| summarize count() by HostIP\n| top 10 by count_ desc"
+        }	
+    ],
+    "dataTypes": [
+        {
+            "name": "Syslog(Firebox)",
```
Comment (nazang) [inline DataConnectors/Connector_syslog_WatchGuardFirebox.json:30]: Syslog (Firebox)  

Make sure there is a space before bracket.

File: DataConnectors/Connector_syslog_WatchGuardFirebox.json [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "Firebox\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
```
Comment (nazang) [inline DataConnectors/Connector_syslog_WatchGuardFirebox.json:42]: Set preview flag for now:

"availability": { 
            "status": 1,
            "isPreview": true
            },

File: DataConnectors/Connector_syslog_WatchGuardFirebox.json [hunk 3]
```diff
@@ -0,0 +1,117 @@
+{
+    "id": "WatchguardFirebox",
+    "title": "Firebox",
```
Comment (nazang) [inline DataConnectors/Connector_syslog_WatchGuardFirebox.json:3]: Title should be Provider Name and Appliance Name:

WatchGuard Firebox

File: DataConnectors/Connector_syslog_WatchGuardFirebox.json [hunk 4]
```diff
@@ -0,0 +1,117 @@
+{
+    "id": "WatchguardFirebox",
+    "title": "Firebox",
+    "publisher": "Watchguard",
```
Comment (nazang) [inline DataConnectors/Connector_syslog_WatchGuardFirebox.json:4]: provide should match the company name in cases:

WatchGuard or WatchGuard Technologies

File: DataConnectors/Connector_syslog_WatchGuardFirebox.json [hunk 5]
```diff
@@ -0,0 +1,117 @@
+{
+    "id": "WatchguardFirebox",
+    "title": "Firebox",
+    "publisher": "Watchguard",
+    "descriptionMarkdown": "Syslog is an event logging protocol that is common to Linux. Applications will send messages that may be stored on the local machine or delivered to a Syslog collector. When the Agent for Linux is installed, it configures the local Syslog daemon to forward messages to the agent. The agent then sends the message to the workspace.\n\n[Learn more >](https://aka.ms/sysLogInfo)",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **<enter the Kusto Function alias>** in queries and workbooks. [Follow steps to get this Kusto Function>](<Link to Kusto Function on Azure Sentinel GitHub>)  ",
+    "graphQueries": [
```
Comment (nazang) [inline DataConnectors/Connector_syslog_WatchGuardFirebox.json:7]: [critical missing parts] Parser should be defined and include the defined alias and link into the subscription.

File: DataConnectors/Connector_syslog_WatchGuardFirebox.json [hunk 6]
```diff
@@ -0,0 +1,117 @@
+{
+    "id": "WatchguardFirebox",
+    "title": "Firebox",
+    "publisher": "Watchguard",
+    "descriptionMarkdown": "Syslog is an event logging protocol that is common to Linux. Applications will send messages that may be stored on the local machine or delivered to a Syslog collector. When the Agent for Linux is installed, it configures the local Syslog daemon to forward messages to the agent. The agent then sends the message to the workspace.\n\n[Learn more >](https://aka.ms/sysLogInfo)",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **<enter the Kusto Function alias>** in queries and workbooks. [Follow steps to get this Kusto Function>](<Link to Kusto Function on Azure Sentinel GitHub>)  ",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Watchguard",
```
Comment (nazang) [inline DataConnectors/Connector_syslog_WatchGuardFirebox.json:10]: legend should be "WatchGuard Firebox"

File: DataConnectors/Connector_syslog_WatchGuardFirebox.json [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+    "id": "WatchguardFirebox",
+    "title": "Firebox",
+    "publisher": "Watchguard",
+    "descriptionMarkdown": "Syslog is an event logging protocol that is common to Linux. Applications will send messages that may be stored on the local machine or delivered to a Syslog collector. When the Agent for Linux is installed, it configures the local Syslog daemon to forward messages to the agent. The agent then sends the message to the workspace.\n\n[Learn more >](https://aka.ms/sysLogInfo)",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **<enter the Kusto Function alias>** in queries and workbooks. [Follow steps to get this Kusto Function>](<Link to Kusto Function on Azure Sentinel GitHub>)  ",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Watchguard",
+            "baseQuery": "Firebox"
```
Comment (nazang) [inline DataConnectors/Connector_syslog_WatchGuardFirebox.json:11]: "baseQuery": "DATATYPE_NAME", Validate DATATYPE_NAME represent both provider, appliance name and type of data [optionally] as a short name in the data type name.

DATATYPE_NAME should be WatchGuardFirebox

File: DataConnectors/Connector_syslog_WatchGuardFirebox.json [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+                    "write": true,
+                    "delete": true
+                }
+            }
+        ]
+    },
+    "instructionSteps": [
+        {
+            "title": "", 
+            "description": ">This data connector depends on a parser based on a Kusto Function to work as expected. [Follow these steps](Link to Kusto Function on Azure Sentinel GitHub) to create the Kusto Functions alias, **enter the Kusto Function alias**",
```
Comment (nazang) [inline DataConnectors/Connector_syslog_WatchGuardFirebox.json:62]: Again, this should have the function alias defined and has a valid link to the function.

**Review by nazang — CHANGES_REQUESTED** [review]

**Pull request closed without merging**